### PR TITLE
fix(formEditor): fix wraps in buttons with large texts. Closes #221

### LINKED ISSUE #221: Button styles differs in the edit mode and in the preview mode in the Bar forms
http://snpy.in/vc9ovC

### PR DESCRIPTION
**Pull request by zouxuoz**
https://github.com/sendsay-ru/account/issues/221

### CODE REVIEW

File: src/blocks/formEditor/formEditorElement/formEditorElement.less [hunk 1]
```diff
@@ -141,12 +141,14 @@
 }
 
 .formEditorElement__button {
+    white-space: nowrap;
     -webkit-user-select: none;
        -moz-user-select: none;
             user-select: none;
     pointer-events: none;
 }
```
Comment (radiansz) [inline src/blocks/formEditor/formEditorElement/formEditorElement.less]: Extra new line

Comment (radiansz) [inline src/blocks/formEditor/formEditorElement/formEditorElement.less]: Extra new line

Comment (zouxuoz) [inline src/blocks/formEditor/formEditorElement/formEditorElement.less]: @radiansz fixed

**Comment by radiansz** [comment]
LGTM :zap:

**Pull request merged**